cartography/intel/gitlab/resources.py

```diff
@@ -0,0 +1,11 @@
+from typing import Dict
+
+from . import members
+from . import projects
+from . import repositories
+
+
+RESOURCE_FUNCTIONS: Dict = {
```

Comment:
I don't see repository. let's add repository as well. That's a key resource for gitlab inventory

Reply:
done